src/reporter/index.js

```diff
@@ -132,8 +132,8 @@ If the source documents are accessible in a browser but fetching them always fai
 ### References
 
 - ${latestDeclarationLink}
-- ${latestVersionLink}
-- ${latestSnapshotsLink}
+${this.repositories.versions ? `- ${latestVersionLink}` : ''}
+${this.repositories.snapshots ? `- ${latestSnapshotsLink}` : ''} 
```

Comment:
I am not sure I understand: what would it mean that only declarations are defined, but not snapshots nor versions? Do we want to support such behaviour? 🤔